src/main/java/uk/ac/cam/cl/dtg/segue/api/EmailFacade.java

```diff
@@ -218,8 +219,9 @@ public final Response getEmailInBrowserById(@Context final HttpServletRequest re
             previewProperties.putAll(emailManager.flattenTokenMap(userPropertiesMap, Maps.newHashMap(), ""));
```

Comment:
The user propertiesMap contain most user fields including givenName and familyName which then get added to the previewProperties without being escaped. So if a user was to use `givenname` it would be escaped, if they were to use `givenName` it would not be escaped.

Reply:
This is also true in the email manager, so when we fix this it would be good to also remove the duplicated logic.

Reply:
It would be nice if we could get rid of the legacy givenname and use the camelcase properties. I remember we added it in as both mainly because we made a mistake once and used givenname. Would require a find and replace over the content repo though.